paymentsheet/src/main/java/com/stripe/android/paymentsheet/PaymentSheetViewModel.kt

```diff
@@ -818,7 +832,13 @@ internal class PaymentSheetViewModel @Inject internal constructor(
         }
         val hasPaymentMethods = !paymentMethods.value.isNullOrEmpty()
         val target = if (hasPaymentMethods) {
-            PaymentSheetScreen.SelectSavedPaymentMethods
+            PaymentSheetScreen.SelectSavedPaymentMethods(
```

Comment:
it's probably worth doing this in the SelectSavedPaymentMethods constructor, so you don't have to duplicate this code

Reply:
I added a util method to BaseSheetViewModel to get the CvcRecollection state to avoid always having to pass in a StateFlow<CvcController> to the screen